### DISCUSSION TITLE
pulling in equity curve

### DISCUSSION THREAD

**Discussion by gaussianswan** [kernc/backtesting.py#214] [Q&A] [answered]
Not sure if this is a dumb question but is it possible to pull in the equity curve after we have run our backtest? Thanks

**Comment by kernc** [comment, marked as answer]
Pull in?
```py
stats = backtest.run(...)
stats._equity_curve.Equity: pd.Series
```
It's right there in the [quick start manual](https://kernc.github.io/backtesting.py/doc/examples/Quick%20Start%20User%20Guide.html#Trade-data). :unamused:

**Reply by gaussianswan** [reply]
i clearly can't read. thanks!
